about.html

```diff
@@ -0,0 +1,191 @@
+                this activity!</i
+              ><br /><b>info@jsociety.org</b>
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about-us flex-column-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <img
+        src="./Assets/images/about us background.PNG"
+        alt="about us community photo"
+        class="family-photo"
+      />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          These seriously fun congresses help students develop and apply a conceptual understanding of science, technology, engineering, and math by solving puzzles, designing solutions to novel problems, and creating new inventions. It's important for students of all ages to engage in this type of critical thinking to prepare for college and career. Check out our list of tools that get students coding robots, engineering new creations, playing with games and simulations, and gathering and analyzing scientific data. 
+        </p>
+        <h4>
+          <br />
+          What do we promise to Javascript enthusiasts?
+        </h4>
+        <p>
+          <br />
+          The nub of the challenge will be to design a new academic day to allow gifted students to comfortably and confidently “linger and learn” at the interfaces between those tectonic plates, exploring and discussing potential new ideas…and learning how to solve problems in a blended, mediated way. Gifted students need time to think and formulate questions, to learn from each other. Educators need to respond to this challenge, to break away from the 140-year old pizza-pie slice model of the academic day. Teacher colleges beware, big change is coming. Can you not feel the winds of change blowing thru the hallways…a breeze perhaps now, but destined soon to be gale force in strength?
+        </p>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <section class="past-activities" id="past-activities">
+      <h2 class="past-heading light-black heading-medium">
+        See the past Community Activities
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <p class="past-content light-black text-center">
+        Take a look at the last two years activities which took place in
+        Nairobi
+      </p>
+      <div class="activities">
+        <div class="past-first-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2021</h3>
+            <p>JSociety impact congress</p>
+          </div>
+        </div>
+        <div class="past-second-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2022</h3>
+            <p>JSociety impact congress</p>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Please decrease the gap between the two event images in about section to match with the provided design


Desktop Version
----
Cindy's | Your's
---|---
<img width="916" alt="Screenshot 2023-07-07 at 11 57 54 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/d8a80eb4-5abf-4fc2-bac3-3431104ac293">|<img width="1368" alt="Screenshot 2023-07-07 at 10 22 01 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/76cacf36-171c-4c8e-bdc7-a4a9a1c6197e">

Mobile Version
----
Cindy's | Your's
---|---
<img width="481" alt="Screenshot 2023-07-07 at 11 57 46 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/80dde69a-3c0e-4054-9ae3-222c41bf5d89">|<img width="365" alt="Screenshot 2023-07-07 at 12 08 47 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/e6fbea1b-40b7-4f09-a251-26b14555df01">


- [ ] Also please add a bit of margin in the mobile version for the "See the past Community Activities" title as it is almost collapsing with the borders and looks cramped.